apps/backend/src/libs/helpers/helpers.ts

```diff
@@ -1,3 +1,4 @@
+export { checkUserPermissions as checkUserPermissionsHelper } from "./check-user-permissions.helper.js";
```

Comment:
Maybe we can name it as checkProjectPermission instead?

Reply:
it checks all permissions

Reply:
it checks all permissions

Reply:
I changed it to `checkPermissions`, because I just replaced the functional from `checkUserPermissions` hook to not copy the code